rust/plugin-lib/src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub trait Plugin {
     /// Called when an edit has occured in the remote view. If the plugin wishes
     /// to add its own edit, it may return `Some(edit)`.
     fn update(&mut self, view: &mut View<Self::Cache>, delta: Option<&RopeDelta>,
-              edit_type: String, author: String) -> Option<PluginEdit>;
+              edit_type: String, author: String) -> Option<u64>;
```

Comment:
This can now return nothing; the plugin lib can handle acking the update without the plugin needing to know about it.